app/components/conferences/admin/editEvent.js

```diff
@@ -0,0 +1,164 @@
+      if (e.response.status == 401) {
+        Cookies.remove("event_auth");
+        router.push("/conferences");
+      }
+      throw new Error(e);
+    }
+  };
+
+  const handleChange = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    setFormState((prev) => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+
+  return (
+    <>
+    <Card>
+      <Card.Header>Editing Event {formState.name}!</Card.Header>
+      <Card.Body>
+        <Form onSubmit={handleFormSubmit}>
+          <Form.Group className="mb-3">
+            <Form.Label>Event name*</Form.Label>
+            <Form.Control
+              required
+              value={formState.name}
+              name="name"
+              type="text"
+              placeholder=""
+              onChange={handleChange}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>Event Description</Form.Label>
+            <Form.Control
+              as="textarea"
+              name="description"
+              type="text"
+              placeholder=""
+              onChange={handleChange}
+              value={formState.description}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>Event Banner Image URI</Form.Label>
+            <Form.Control
+              name="original-image-url"
+              type="url"
+              placeholder="https://via.placeholder.com/1920x960.png"
+              onChange={handleChange}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>Event Organizer logo</Form.Label>
+            <Form.Control
+              name="logo-url"
+              type="url"
+              placeholder="https://via.placeholder.com/100.png"
+              onChange={handleChange}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>Start Date*</Form.Label>
+            <Form.Control
+              required
+              name="starts-at"
+              type="datetime-local"
+              value={formState["starts-at"]}
+              min={formState["start-at"]}
+              onChange={handleChange}
+              placeholder=""
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>End Date*</Form.Label>
+            <Form.Control
+              required
+              name="ends-at"
+              type="datetime-local"
+              value={formState["ends-at"]}
+              min={formState["start-at"]}
+              onChange={handleChange}
+              placeholder=""
+            />
+          </Form.Group>
+          <ButtonGroup aria-label="Basic example">
+            <Button variant="primary" type="submit">
+              Save
+            </Button>
+          </ButtonGroup>
+        </Form>
+      </Card.Body>
+    </Card>
+    </>
+
+  );
```

Comment:
@ayush3160 thanks for the PR. Can you come up with a solution to avoid re-writing the same code as in the `EventBasicCreate` component? 
An idea would be:
Since the only differences between the two components (`EventBasicCreate` and `EditEvent`) are:
1. Card Header
2. Functions to handle form submit
So, what you can do is pass all this as `props` to the `EventBasicCreate` or create a new component, `EventForm` which will render the `return` (line 73 - line 152).

If you have any further doubts, please feel free to ask on RC or here.
Thank you!